File: gtn.py
# GUESS THE NUMBER
# 25/10/21
# BY REMTII
import random

x = random.randrange(1, 20)
win = False
print("Random number betweens 1 and 20")
while win == False:
    y = int(input("=] "))
    if y > x:
        print("Too big")
    elif y < x:
        print("Too small")
    elif y == x:
        print("You win! :D")
        win = True

File: gtn.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int rndRange(int min, int max) {
	return (rand() % (max - min + 1)) + 1;
}

int main() {
	srand(time(0));
	int x = rndRange(1, 20);
	int y;
	int win = 0;
	printf("Guess the number betweens 1 and 20\n");
	while (win == 0) {
		printf("=] ");
		scanf("%d", &y);
		if (y < x) {
			printf("Too small\n");
		} else if (y > x) {
			printf("Too big\n");
		} else if (y == x) {
			printf("You win! :D\n");
			win = 1;
		}
	}

	return 0;
}
